server/src/main/java/org/apache/iotdb/db/metadata/idtable/AppendOnlyDiskSchemaManager.java

```diff
@@ -54,12 +58,15 @@ public class AppendOnlyDiskSchemaManager implements IDiskSchemaManager {
 
   long loc;
 
+  private RandomAccessFile randomAccessFile;
+
   private static final Logger logger = LoggerFactory.getLogger(AppendOnlyDiskSchemaManager.class);
 
   public AppendOnlyDiskSchemaManager(File dir) {
     try {
       initFile(dir);
       outputStream = new FileOutputStream(dataFile, true);
+      randomAccessFile = new RandomAccessFile(dataFile, "rw");
```

Comment:
Remember close this file when the manager is closed or cleared. Otherwise, some IT may be affected, since the file handler protection on windows is strict.

Reply:
I close randomAccessFile in the close method of this class